### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by smanish1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # dll
 doubly linked list javascript class
+test
```
Comment (smanish01) [inline README.md]: test comment in pull request

Comment (smanish01) [inline README.md]: test comment in pull request

Comment (smanish01) [inline README.md]: test comment pull request 2

Comment (smanish1) [inline README.md]: test

Comment (smanish1) [inline README.md]: cdxcdx

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # dll
 doubly linked list javascript class
```
Comment (smanish01) [inline README.md]: fvdg

**Pull request merged**